app/bundles/CoreBundle/Helper/BuilderTokenHelper.php

```diff
@@ -94,7 +94,8 @@ public function getTokens(
         }
 
         $exprBuilder = $this->connection->createExpressionBuilder();
-
+        $expr = $expr ?? $this->connection->createExpressionBuilder();
```

Comment:
Tests uncovered it's not working:
```
TypeError: Mautic\CoreBundle\Entity\CommonRepository::getSimpleList(): Argument #1 ($expr) must be of type ?Doctrine\DBAL\Query\Expression\CompositeExpression, Doctrine\DBAL\Query\Expression\ExpressionBuilder given, called in /home/runner/work/mautic/mautic/app/bundles/CoreBundle/Helper/BuilderTokenHelper.php on line 117
```
How about something like
```suggestion
        $expr = $expr ?? $this->connection->createQueryBuilder()->expr();
```